WalletWasabi.Fluent/Views/Wallets/WalletView.axaml

```diff
@@ -10,46 +10,70 @@
-                          ScrollViewer.VerticalScrollBarVisibility="Disabled">
-                <ItemsRepeater Items="{Binding Tiles}"
-                               DockPanel.Dock="Top"
-                               Margin="0 0"
-                               HorizontalAlignment="Left"
-                               VerticalAlignment="Top"
-                               UseLayoutRounding="False">
-                    <ItemsRepeater.Styles>
-                        <Style Selector="ItemsRepeater > :is(Control)" x:DataType="tiles:TileViewModel">
-                            <Setter Property="(c:NonVirtualizingResponsiveLayout.ColumnSpan)"
-                                    Value="{Binding ColumnSpan}" />
-                            <Setter Property="(c:NonVirtualizingResponsiveLayout.RowSpan)" Value="{Binding RowSpan}" />
-                        </Style>
-                    </ItemsRepeater.Styles>
-                    <ItemsRepeater.ItemTemplate>
-                        <locator:ViewLocator />
-                    </ItemsRepeater.ItemTemplate>
-                    <ItemsRepeater.Layout>
-                        <c:NonVirtualizingResponsiveLayout ItemWidth="330"
-                                                           ItemHeight="150"
-                                                           WidthSource="{Binding #ScrollViewer.Bounds.Width}"
-                                                           AspectRatio="NaN"
-                                                           ColumnHints="3, 3, 3"
-                                                           WidthTriggers="300, 660, 990" />
-                    </ItemsRepeater.Layout>
-                </ItemsRepeater>
-
-            </ScrollViewer>
-            <history:HistoryTable Name="HistoryTable" DataContext="{Binding History}" MaxWidth="1000"
-                                  HorizontalAlignment="Left" VerticalAlignment="Top" />
+  <c:ContentArea ScrollViewer.VerticalScrollBarVisibility="Disabled" ClipToBounds="False">
+    <c:ContentArea.Title>
+      <StackPanel>
+        <DockPanel>
+          <TextBlock Text="{Binding WalletName, FallbackValue=My Wallet}" />
+          <StackPanel HorizontalAlignment="Right" Margin="0 -5 0 0" DockPanel.Dock="Right" Orientation="Horizontal" Spacing="10">
+            <Button Classes="function" Command="{Binding SendCommand}">
+              <StackPanel Orientation="Horizontal" Spacing="15">
+                <PathIcon Data="{StaticResource wallet_action_send}" />
+                <TextBlock Text="Send" />
+              </StackPanel>
+            </Button>
+            <Button Classes="function" Command="{Binding ReceiveCommand}">
+              <StackPanel Orientation="Horizontal" Spacing="15">
+                <PathIcon Data="{StaticResource wallet_action_receive}" />
+                <TextBlock Text="Receive" />
+              </StackPanel>
+            </Button>
+          </StackPanel>
```

Comment:
We need a similar solution here as we have at the ActionCenter's search bar.
![image](https://user-images.githubusercontent.com/16364053/117951711-b5751880-b314-11eb-825b-51bad70269e0.png)

Reply:
solved with text trimming.